modules/update-mirror-repository.adoc

```diff
@@ -3,10 +3,8 @@
 // * updating/updating-restricted-network-cluster.adoc
 
 :_content-type: PROCEDURE
-[id="update-mirror-repository_{context}"]
-= Mirroring the {product-title} image repository
-
-Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+[id="update-mirror-repository_adm_release_mirror_{context}"]
+= Mirroring images using the adm release mirror command
 
```

Comment:
nit: this section will have installing `oc` as a prereq.  Not sure if it needs to get listed again here, or if we can inherit it from some wrapping context/module.